ethstats/ethstats.go

```diff
@@ -244,20 +262,20 @@ func (s *Service) readLoop(conn *websocket.Conn, in *json.Decoder) {
 			// Make sure the request is valid and doesn't crash us
 			request, ok := msg["emit"][1].(map[string]interface{})
 			if !ok {
-				log.Warn("Invalid history request", "msg", msg["emit"][1])
-				return
+				log.Warn("Invalid stats history request", "msg", msg["emit"][1])
```

Comment:
I think it also has to do something with "trusted" vs. "untrusted" nodes within the stats tracking in ethstats. Not sure tbh, the code is quite messy and at this point I'm happy if we can handle issues gracefully that trying to understand what fails. Marian started working on a new page afaik a while ago, so bugs won't really get fixed at this point.